finality-aleph/src/sync/mock/mod.rs

```diff
@@ -93,6 +93,38 @@ impl Header for MockHeader {
     }
 }
 
+#[derive(Clone, Hash, Debug, PartialEq, Eq, Encode, Decode)]
+pub struct MockBlock {
+    header: MockHeader,
+}
+
+impl MockBlock {
+    #[allow(dead_code)]
+    pub fn for_header(header: MockHeader) -> Self {
+        Self { header }
+    }
+}
+
+impl Header for MockBlock {
```

Comment:
MockBlock -> MockHeader?

Reply:
nah, I want the `MockBlock` to have `id` and `parent_id`, similar to `impl Header for MockJustification { ... ` in line 143